source/fundamentals/enterprise-auth.txt

```diff
@@ -150,12 +153,42 @@ You may need to specify one or more of the following additional
       - :java-docs:`CANONICALIZE_HOST_NAME_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#CANONICALIZE_HOST_NAME_KEY>`
       - :java-docs:`JAVA_SUBJECT_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_KEY>`
       - :java-docs:`JAVA_SASL_CLIENT_PROPERTIES_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SASL_CLIENT_PROPERTIES_KEY>`
+      - :java-docs:`JAVA_SUBJECT_PROVIDER_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_PROVIDER_KEY>`
 
       Your code to instantiate a ``MongoClient`` using GSSAPI and additional
       properties might resemble the following:
 
       .. include:: /includes/fundamentals/code-snippets/auth-credentials-gssapi-properties.rst
 
+
+The Java Driver will by default cache Kerberos tickets by ``MongoClient`` instance.
+If your deployment needs to frequently create and destroy ``MongoClient`` instances, 
+you can change the default Kerberos ticket caching behavior of the Java Driver
+to cache by process.
```

Comment:
```suggestion
you can change the default Kerberos ticket caching behavior to cache by process.
```

"The Java Driver" is already mentioned in the previous sentence

Reply:
Thanks for the note. I'll take it out. I think you are correct in saying that there is enough context that it does not to be mentioned again.